lib/load.php

```diff
@@ -103,6 +103,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 
 // Include FSE related files only if the experiment is enabled.
 if ( gutenberg_is_experiment_enabled( 'gutenberg-full-site-editing' ) ) {
+	require dirname( __FILE__ ) . '/full-site-editing/templates-definitions.php';
```

Comment:
We are starting to have a lot of FSE-related library files, so I think it's the case to group them in a separate folder.
I haven't moved the others to avoid bloating this PR.